<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="HP" name="appiumdemo.RegisterLazada" tests="15" failures="1" timestamp="13 Aug 2017 09:24:08 GMT" time="160.550" errors="0">
  <testcase name="validateEmail" time="11.125" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validatePassword" time="10.736" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validatePassword" time="10.916" classname="appiumdemo.RegisterLazada"/>
  <testcase name="testFullName" time="14.332" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validatePassword" time="10.334" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validatePassword" time="10.745" classname="appiumdemo.RegisterLazada">
    <failure type="java.lang.AssertionError" message="expected [Mật khẩu phải chứa ít nhất một chữ và số.] but found [Mật khẩu phải chứa ít nhất một chữ và số]">
      <![CDATA[java.lang.AssertionError: expected [Mật khẩu phải chứa ít nhất một chữ và số.] but found [Mật khẩu phải chứa ít nhất một chữ và số]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at appiumdemo.RegisterLazada.validatePassword(RegisterLazada.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- validatePassword -->
  <testcase name="validateEmail" time="11.116" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validateEmail" time="10.571" classname="appiumdemo.RegisterLazada"/>
  <testcase name="testFullName" time="13.097" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validateEmail" time="7.859" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validatePassword" time="10.737" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validateEmail" time="10.217" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validatePassword" time="7.786" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validateEmail" time="10.357" classname="appiumdemo.RegisterLazada"/>
  <testcase name="validateEmail" time="10.622" classname="appiumdemo.RegisterLazada"/>
</testsuite> <!-- appiumdemo.RegisterLazada -->
